src/cljc/athens/common_events/resolver/atomic.cljc

```diff
@@ -34,7 +34,9 @@
                                         (if ref-block-exists?
                                           ref-block
                                           (throw (ex-info "Ref block does not exist" {:block/uid ref-uid})))
-                                        (common-db/get-parent db [:block/uid ref-uid]))
+                                        (if-let [parent (common-db/get-parent db [:block/uid ref-uid])]
+                                          parent
+                                          (throw (ex-info "Ref block does not have parent" {:block/uid ref-uid}))))
```

Comment:
What has happened here? Why did we start throwing this error here in this PR?
Also no test reflecting this change in behaviour.

Same comment for `:block/move`.

Reply:
While replaying the team graph log I saw a nullpointer error here (specifically, when trying to reorder children if there's no parent). The nullpointer error was not caught by our current trycatch and would crash the process. 

I think the right thing is to throw a legible error here. You're right that there's no tests and there should be. Adding now.